Bottom: 4b3500a8fd17479afa2b5fcecb24ab30efb24c78
Top:    974ed983ca74e2a55568d7f754dc3f1360e67a2f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 00:22:47 +0100

print.c: Fixup warnings due to assigned but unused variables


---

diff --git a/src/print.c b/src/print.c
index 1e3abc0..7e7e412 100644
--- a/src/print.c
+++ b/src/print.c
@@ -222,12 +222,10 @@ PrintFab_overhang (void)
 void
 PrintFab (hidGC gc)
 {
-  PinType tmp_pin;
   DrillInfoTypePtr AllDrills;
   int i, n, yoff, total_drills = 0, ds = 0;
   time_t currenttime;
   char utcTime[64];
-  tmp_pin.Flags = NoFlags ();
   AllDrills = GetDrillInfo (PCB->Data);
   RoundDrillInfo (AllDrills, MIL_TO_COORD(1));
   yoff = -TEXT_LINE;
